src/util/thpool/thpool.c

```
@@ -325,11 +325,11 @@ void thpool_get_tasks
 	job *job = jobqueue_p->front;
 
 	// iterate through job queue
-	uint32_t i = 0;
-	for(; i < jobqueue_p->len && i < *num_tasks; i++) {
+	uint32_t k = 0;  // number of matched tasks
+	for(uint32_t i; i < jobqueue_p->len && i < *num_tasks; i++) {
```

cppcheck (2.10-2)

[warning] Uninitialized variable

(uninitvar)